### PR TITLE
LinkedList Complete

### PR DESCRIPTION
**Pull request by Alibabji**
Added LinkedList class for all data type in c++

Added Functions for LinkedList class:
- `pushFront()`
- `pushBack()`
- `insert()`
- `popFront()`
- `popBack()`
- `remove()`
- `swap()`
- `get()`
- `output()`
- `isEmpty()`
- `length()`

### CODE REVIEW

File: Data_Structure_Study/LinkedList.cpp [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+template <typename T>
+int LinkedList<T>::length()
+{
+	return size;
+}
+template <typename T>
+bool LinkedList<T>::isEmpty()
+{
+	return (size ? false : true);
+}
```
Comment (uplsh580) [inline Data_Structure_Study/LinkedList.cpp]: new line

File: Data_Structure_Study/LinkedList.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	T popBack();
+	void remove(int pos);
+	void swap(int pos1, int pos2);
+	T get(int pos);
+	void output();
+	bool isEmpty();
+	int length();
+};
+
+#endif
```
Comment (uplsh580) [inline Data_Structure_Study/LinkedList.h]: new line

File: Data_Structure_Study/Main.cpp [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    cout << endl;
+    cout << endl;
+
+	LinkedList<double> list2;
+	list2.pushBack(1.234);
+	cout <<endl<< list2.length() << endl << list2.isEmpty() << endl << list.length() << endl << list.isEmpty();
+
+	cout << endl << endl << endl << endl << list.get(1);
+    return 0;
+}
```
Comment (uplsh580) [inline Data_Structure_Study/Main.cpp]: new line

Comment (Alibabji) [reply Data_Structure_Study/Main.cpp]: 반영했습니다

File: Data_Structure_Study/LinkedList.h (hunk 2 above)
Comment (Alibabji) [reply Data_Structure_Study/LinkedList.h]: 반영했습니다

File: Data_Structure_Study/LinkedList.cpp (hunk 1 above)
Comment (Alibabji) [reply Data_Structure_Study/LinkedList.cpp]: 반영했습니다

**Review by uplsh580 — APPROVED** [review]
LGTM 👍

**Pull request merged**